.changeset/many-trees-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/config-loader': patch
```

Comment:
Breaking changes need a `minor` bump, which likely applies to all of these changesets. Only exception is probably `app-defaults`, which I don't think has a breaking change